### DISCUSSION TITLE
Planning to move this repository to the xpublish-community organization

### DISCUSSION THREAD

**Discussion by jhamman** [xarray-contrib/xpublish#181] [General]
Per the discussion at [today's community meeting](https://hackmd.io/8ipG-H0lRWqDYd-2dOJ43w?view), the plan is to move this repository to the [xpublish-community organization](https://github.com/xpublish-community). That new organization will be a home for Xpublish and a growing collection of community developed plugins. I'll plan to move the repo in a few days unless I hear any objections.

cc @xarray-contrib/xpublish-devs

**Comment by abkfenris** [comment]
I've moved my EDR and OpenDAP plugins over to xpublish-community, and I'll work on shared community resources (code of conduct, single discussion, ...). 

I'll also work on setting up an 'xpublish-experiments' organization for things that are more to show off, but aren't going to be maintained.

**Comment by abkfenris** [comment]
@jhamman can you make sure the secrets transfer over so we can cut a release afterwards?

xref: #183

**Reply by jhamman** [reply]
Done!

**Reply by abkfenris** [reply]
@jhamman It looks like the permissions on the repo were reset with the move. Can you take a look at that?

**Reply by jhamman** [reply]
Should be fixed now

![image](https://user-images.githubusercontent.com/2443309/231209726-ff1ee228-6f9c-447c-b8b8-e688d3558024.png)

**Reply by benbovy** [reply]
Should we also add to `@xpublish-community/core-maintainers` the members missing from `@xarray-contrib/xpublish-devs`? Although I've not been very active recently here, I'm very happy to see you pushing this project further and I'd be glad if I could keep being involved or help somehow ;-).

**Reply by jhamman** [reply]
All previous members have now been moved over to this team.

**Reply by benbovy** [reply]
Thanks Joe!

**Comment by mpiannucci** [comment]
I moved over xpublish-wms but realized today that I can't push to it anymore. Can I get the permissions to maintain that repo?

**Reply by abkfenris** [reply]
I think this will take @jhamman as I don't have enough permission to make changes. At a quick look I also can't push to EDR or OpenDAP plugins myself.

**Reply by mpiannucci** [reply]
Just bumping this :)

**Reply by jhamman** [reply]
this is done.

**Reply by mpiannucci** [reply]
Thank you!!!
